package.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<package>
+  <name>wm_kinova_hardware_interface</name>
+  <version>0.0.0</version>
+  <description>The sara_arm package</description>
+  <maintainer email="philippe.la-madeleine.1@ens.etsmtl.ca">PhilLM</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```

Comment:
Add a license please!
Also, I think you could change the version (`0.1.0` or `1.0.0`)